src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogDto.cs

```diff
@@ -12,155 +12,536 @@ namespace Digdir.Domain.Dialogporten.Application.Features.V1.ServiceOwner.Dialog
+    /// </summary>
+    /// <example>urn:altinn:resource:some-service-identifier</example>
     public string ServiceResource { get; set; } = null!;
+
+    /// <summary>
+    /// The party code representing the organization or person that the dialog belongs to in URN format
+    /// </summary>
+    /// <example>
+    /// urn:altinn:person:identifier-no:01125512345
+    /// urn:altinn:organization:identifier-no:912345678
+    /// </example>
     public string Party { get; set; } = null!;
+
+    /// <summary>
+    /// Advisory indicator of progress, represented as 1-100 percentage value. 100% representing a dialog that has come
+    /// to a natural completion (successful or not).
+    /// </summary>
     public int? Progress { get; set; }
+
+    /// <summary>
+    /// Arbitrary string with a service-specific indicator of status, typically used to indicate a fine-grained state of
+    /// the dialog to further specify the "status" enum.
+    /// </summary>
     public string? ExtendedStatus { get; set; }
+
+    /// <summary>
+    /// Arbitrary string with a service-specific reference to an external system or service.
+    /// </summary>
     public string? ExternalReference { get; set; }
+
+    /// <summary>
+    /// The timestamp when the dialog should be made visible for authorized end users. If not provided, the dialog will be
+    /// immediately available.
+    /// </summary>
+    /// <example>2022-12-31T23:59:59Z</example>
     public DateTimeOffset? VisibleFrom { get; set; }
+
+    /// <summary>
+    /// The due date for the dialog. Dialogs past due date might be marked as such in frontends but will still be available.
+    /// </summary>
+    /// <example>2022-12-31T23:59:59Z</example>
     public DateTimeOffset? DueAt { get; set; }
+
+    /// <summary>
+    /// The expiration date for the dialog. This is the last date when the dialog is available for the end user.
+    ///
+    /// After this date is passed, the dialog will be considered expired and no longer available for the end user in any
+    /// API. If not supplied, the dialog will be considered to never expire. This field can be changed after creation.
+    /// </summary>
+    /// <example>2022-12-31T23:59:59Z</example>
     public DateTimeOffset? ExpiresAt { get; set; }
 
+    /// <summary>
+    /// If set, will override the date and time when the dialog is set as created
+    /// If not supplied, the current date /time will be used.
+    /// </summary>
+    /// <example>2022-12-31T23:59:59Z</example>
+    public DateTimeOffset CreatedAt { get; set; }
+
+    /// <summary>
+    /// If set, will override the date and time when the dialog is set as last updated
+    /// If not supplied, the current date /time will be used.
+    /// </summary>
+    /// <example>2022-12-31T23:59:59Z</example>
+    public DateTimeOffset UpdatedAt { get; set; } // TODO: This does not work, as UpdatedAt is always overwritten
```

Comment:
We need to be able to take both predefined CreatedAt and UpdatedAt values in CreateDialog in order to support import scenarios. However, this will require more than a trivial change as IUpdatable entities always gets its UpdateAt field populated with current timestamp. Suggest we leave this for this PR and make an issue for addressing that.

Reply:
Hmm - Tar veldig gjerne en liten diskusjon om det er verdt å åpne opp for å sette UpdatedAt. 🙂

Reply:
Dialog list views in AF/other frontends will most likely be sorted descending by "UpdatedAt", not "CreatedAt". Not being able to define UpdatedAt will make archive imports from Altinn 2/3 very difficult, as whatever is imported last ends up on top in any frontend views - regardless of the actual age of the imported item.

Reply:
Lukker denne her, så får vi forsette i #1003